cns/restserver/ipam.go

```diff
@@ -105,7 +185,7 @@ func (service *HTTPRestService) setIPConfigAsAvailable(ipconfig *cns.ContainerIP
 	ipconfig.State = cns.Available
 	ipconfig.OrchestratorContext = nil
 	service.PodIPConfigState[ipconfig.ID] = ipconfig
-	service.PodIPIDByOrchestratorContext[podInfo.GetOrchestratorContextKey()] = ""
+	delete(service.PodIPIDByOrchestratorContext, podInfo.GetOrchestratorContextKey())
```

Comment:
Not for this line particularly, but the comment for this function and the one above it both say ".. takes a lock of the service ... does not take a lock", is that  a typo or is there some part that does take a lock and another part that doesn't?

Reply:
the private method doesn't take a lock, but the exported ones that use this one do. Latest comment specifies does not take lock